src/doc2dash/parsers/utils.py

```diff
@@ -108,7 +109,9 @@ def patch_anchors(parser, show_progressbar):
 
     def patch_files(files):
         for fname, entries in files:
-            full_path = os.path.join(parser.doc_path, fname)
+            full_path = urllib.parse.unquote(
```

Comment:
Given that fname is coming from the index, wouldn't it be safer to only unquote it and not append it to the full path first?

Reply:
You're right. I left like that because I had an error in other place and thought that `fname` maybe wasn't a `str` 😅. I just pushed the change.